.gitignore

```diff
@@ -9,3 +9,4 @@ coverage/
 dump.rdb
 doc/
 docs/
+.*
```

Comment:
Some dotfiles like `.rspec` would be required to be committed to the repository. So, maybe ignoring ALL dotfiles with`.*` is probably not a good idea.

Reply:
I forgot to unstage this file.. Sorry for that..